config/routes.rb

```diff
@@ -63,13 +63,14 @@
   namespace :merchant do
     get '/dashboard', to: 'dashboard#show'
     get '/items', to: 'items#index'
+    get '/orders/:id', to: 'orders#show'
   end
-  
+
   #admin
   namespace :admin do
     get "/dashboard", to: "dashboard#show"
     get "/users/:user_id", to: "users#show"
     resource :orders, only: [:update]
```

Comment:
good use of resource here, steven and i did all of the merchant items crud and didn't use resources, we'll have to refactor that, this looks super clean!